contracts/network/BancorNetwork.sol

```diff
@@ -579,6 +607,41 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
         return _withdrawBaseToken(contextId, provider, completedRequest);
     }
 
+    /**
+     * @inheritdoc IBancorNetwork
+     */
+    function withdrawSurplusTokens(Token pool) external whenNotPaused nonReentrant returns (uint256) {
+        // verify pool collection exists and retrieve it
+        IPoolCollection poolCollection = _poolCollection(pool);
+        // verify trading is disabled for pool
+        if (poolCollection.tradingEnabled(pool)) {
+            revert PoolTradingShouldBeDisabled();
```

Comment:
Should deposits be disabled as well?

Reply:
It’s not a requirement, and it doesn’t affect the POL once trading is disabled

Reply:
👍